src/librustdoc/clean/types.rs

```diff
@@ -175,9 +175,11 @@ impl Item {
     }
 
     crate fn is_crate(&self) -> bool {
-        matches!(*self.kind,
+        matches!(
+            *self.kind,
             StrippedItem(box ModuleItem(Module { is_crate: true, .. }))
-            | ModuleItem(Module { is_crate: true, .. }))
+                | ModuleItem(Module { is_crate: true, .. })
```

Comment:
Why did you indent this? I think the standard style is no indentation on lines beginning with an or-pattern.

Reply:
I simply ran `cargo fmt` after I ended my changes. So formatting questions have to be addressed to them directly. 😆

Reply:
Do you mean `./x.py fmt`? It seems weird that it would change that...

Reply:
I do it a bit differently: `cd src/librustdoc && cargo fmt`. Generally my `rustfmt` version is newer.

Reply:
This came up recently in another PR, explained in detail [here](https://github.com/rust-lang/rust/pull/80843#discussion_r554490965) for anyone interested

Reply:
I see, that's somewhat unfortunate. Though I guess some formatting is better than no formatting :)

By the way, I want to say that I really appreciate rustfmt; it's the best autoformatter I've ever used.

Reply:
> By the way, I want to say that I really appreciate rustfmt; it's the best autoformatter I've ever used.

That's great to hear, thanks for sharing :heart: